libs/fairness/src/lib/Controls/CalloutHelpBar.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IBounds } from "@responsible-ai/core-ui";
+import _ from "lodash";
+import { Stack, Toggle } from "office-ui-fabric-react";
+import React from "react";
+
+import { IErrorPickerPropsV2 } from "../FairnessWizard";
+import { SharedStyles } from "../Shared.styles";
+
+import { CalloutErrorBars } from "./CalloutErrorBars";
+import { CalloutGraph } from "./CalloutGraph";
+
+interface ICalloutHelpBarProps {
+  graphCalloutStrings: string[];
+  fairnessBounds?: Array<IBounds | undefined>;
+  performanceBounds?: Array<IBounds | undefined>;
+  outcomeBounds?: Array<IBounds | undefined>;
+  falsePositiveBounds?: Array<IBounds | undefined>;
+  falseNegativeBounds?: Array<IBounds | undefined>;
+  errorPickerProps: IErrorPickerPropsV2;
+  parentErrorChanged: {
+    (event: React.MouseEvent<HTMLElement>, checked?: boolean): void;
+  };
+}
+
+export class CalloutHelpBar extends React.PureComponent<ICalloutHelpBarProps> {
+  public render(): React.ReactNode {
+    const sharedStyles = SharedStyles();
+
+    const graphInfoButtonId = "graphInfoButtonId";
+    const errorBarsInfoButtonId = "errorBarInfoButtonId";
+
+    return (
+      <Stack horizontal className={sharedStyles.calloutBarWrapper}>
+        <CalloutGraph
+          graphInfoButtonId={graphInfoButtonId}
+          graphCalloutStrings={this.props.graphCalloutStrings}
+        />
+        <Stack className={sharedStyles.errorBarsCalloutWrapper}>
+          <CalloutErrorBars errorBarsInfoButtonId={errorBarsInfoButtonId} />
+          <Toggle
+            className={sharedStyles.toggle}
+            id="errorMetricDropdown"
+            defaultChecked={
+              this.props.errorPickerProps.selectedErrorKey === "enabled"
+            }
+            disabled={
+              (typeof this.props.fairnessBounds === "undefined" ||
+                _.isEmpty(this.props.fairnessBounds.filter(Boolean))) &&
```

Comment:
What does this actually do? I've never seen `filter.(Boolean)` before.

Reply:
filters for truthy values